### PR TITLE
Day 4, tic tac toe with gosu, iteration 1

### PR DESCRIPTION
**Pull request by chitankensuru**
Gonna continue on day 5 to work on this tic tac toe tutorial

### CODE REVIEW

**Comment by patmaddox** [comment]
Go you!

File: 2_adventures/001/chitankensuru/04_gosu_tictactoe/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#Gosu try
+
+Just following this guy's video tutorial to make tic tac toe in ruby with gosu
+https://www.youtube.com/watch?v=tQjjHt0I3_o
+
+Extra things learned today
+.git/info/exclude works similar like .gitignore except it won' be part of a
```
Comment (kotp) [inline 2_adventures/001/chitankensuru/04_gosu_tictactoe/README.md:7]: Funny, I was reading the same thing yesterday, with "4 Ways to Ignore/Exclude Files in Git Repositories".

**Comment by patmaddox** [comment]
I did not know about `.git/info/exclude` - nice!

**Comment by kotp** [comment]
Is anyone using the commit hooks?  

They are similar in that they live in the repository folder `.git/` and don't really affect anyone else.  I have some that I use routinely.  They do different things like when I do commits there is a recording of happy people cheering and clapping, but only when there is adherence to the style guides, which is another hook.

**Comment by chitankensuru** [comment]
Very interesting. I didn't know about commit hooks. Googled some and found a tutorial for a pre-commit hook that runs rubocop on the code. Might as well make that one of my future challenges.

**Comment by kotp** [comment]
Yes... I use rubocop as well.

**Comment by RobAWilkinson** [comment]
I've used commit hooks but only on the github end so when you push to the repo it notifys slack and trello, interesting I should see if I can configure it locally so I don't have to worry when using bitbucket

**Pull request merged**